playbooks/responses/blocked-needs-repro.md

```diff
@@ -2,24 +2,25 @@
 
 Thank you for taking the time to report this issue and helping to make Electron better.
 
-Would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) for a small app that reproduces the issue by itself?
+Would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) to make a stand-alone app that reproduces this issue?
 
 Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.
 
-I'm adding the `blocked/needs-repro` label for this reason. After you've responded, please link to the test case and @ me in a followup comment.
+I'm adding the `blocked/needs-repro` label for this reason. After you make a test case (a [gist](https://gist.github.com) is preferred since gists can be loaded directly into [Electron Fiddle](https://www.electronjs.org/fiddle) for testing), please link to it in a followup comment.
 
 Thanks in advance! Your help is appreciated.
 
 ## If Repro Requires Third-Party Tooling
 
 Thank you for taking the time to report this issue and helping to make Electron better.
 
-Researching issues that require triaging third-party code is usually not feasable due to the high volume of issues that are reported to Electron.
-Would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) for a small app that reproduces the issue by itself without third-party code?
+Researching issues that depend on third-party code is usually not feasible for the Electron team due to the time that would be required to research reported issues with open-ended dependencies.
+
+Would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) for a small app that reproduces the issue by itself without other code?
 
 Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.
 
-I'm adding the `blocked/needs-repro` label for this reason. After you've responded, please link to the test case and @ me in a followup comment.
+I'm adding the `blocked/needs-repro` label for this reason. After you make a test case (a [gist](https://gist.github.com) is preferred since gists can be loaded directly into [Electron Fiddle](https://www.electronjs.org/fiddle) for testing), please link to it in a followup comment.
```

Comment:
if we're asking users to fork quick-start, it's somewhat confusing to ask them to make a gist in the same response - we should probably stick to just one

Reply:
Hmm, I agree we don't want to make it confusing.

I _really_ like the idea of pushing users towards gists for their repro code since it is just so _efficient_ for testing... and I also like the idea of providing a good starting point a la quick-start to make the repro cases more uniform.

I'm wondering if there is some easy-to-understand way to do both of these things?